WordPress/Classes/Models/Blog+Creation.swift

```diff
@@ -0,0 +1,37 @@
+extension Blog {
+
+    /// Creates a blank `Blog` object
+    @objc(createWithAccount:orContext:)
+    static func createBlog(with account: WPAccount?, or context: NSManagedObjectContext) -> Blog {
+        let blog = createBlog(in: account?.managedObjectContext ?? context)
```

Comment:
When seeing this code in use below, I thought `orContext` was a typo. Without being familiar with the objects at play here, I did not expect `WPAccount` to carry its own `managedObjectContext` and thought that `orContext` was meant to be `inContext`, as in "create a blog with the given account and put it in the given context."

Looking at the code, it's clear that's not the case.

I think for me it would have been clear if the method had a signature such as

```swift
@objc(createWithAccount:fallingBackToContext:)
static func createBlog(with account: WPAccount?, fallingBackTo context: NSManagedObjectContext) -> Blog {
```

But ☝️ , it might just be me, and I'd love to hear someone else's opinion on this. _Not_ a blocker 😄

Reply:
I just realized there is only one place that uses this function, so I got rid of it in https://github.com/wordpress-mobile/WordPress-iOS/pull/19925/commits/958fcc84ec05b415501e0d7f4486de19c4bec14f 😄

Reply:
🎉 🧹